CONTRIBUTING.md

```diff
@@ -1,78 +1,158 @@
 # Contributing to Arkouda
 
-Arkouda welcomes contributions via feedback, bug reports, and pull requests.
-
-We use a simple [Git Feature Branch Workflow](https://www.atlassian.com/git/tutorials/comparing-workflows/feature-branch-workflow)
-
-## Development for Arkouda
+Arkouda is an open source project and we love to see new contributors! We welcome contributions via issues and pull requests
+
+<a id="toc"></a>
+## Table of Contents
+
+1. [Adding Issues](#issues)
+   - [Bug Reports](#bug-reports)
+   - [Feature Requests](#feature-requests)
+2. [Developing Arkouda](#development)
+   - [Coding Conventions and Linting](#code-quality)
+   - [Testing](#testing)
+   - [Writing Pull Requests](#writing-prs)
+3. [Core Development Team Only](#core-dev)
+   - [Merging Pull Requests](#merging-prs)
+   - [Release Process](#release)
+
+<a id="issues"></a>
+## Adding Issues <sup><sup><sub><a href="#toc">toc</a></sub></sup></sup>
+Most issues fall under the broad categories of bug reports or feature requests.
+If your issue doesn't fit either of these, please add it anyway and provide as much detail as possible
+
+Using github markdown (especially [code blocks](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks))
+is very appreciated
+
+<a id="bug-reports"></a>
+### Bug Reports <sup><sup><sub><a href="#toc">toc</a></sub></sup></sup>
+- Summary of Problem
+  - What behavior did you observe when encountering the bug?
+  - What behavior did you expect to observe?
+  - Is this a blocking issue with no known work-arounds?
+- Steps to Reproduce
+  - Please provide code that will reproduce the problem if possible.
+Providing simplified programs demonstrating the problem will be appreciated
+- Configuration Information
+  - What's the output of `ak.get_config()['arkoudaVersion']`
+
+<a id="feature-requests"></a>
+### Feature Requests <sup><sup><sub><a href="#toc">toc</a></sub></sup></sup>
+Be as specific as possible and provide examples when appropriate. If the requested feature is based on another library 
+(i.e. numpy, pandas, scipy), add a link to their documentation on it
```

Comment:
Maybe update `add a link to their documentation on it` to `please provide a link to the supporting documentation for reference.` I think this is a little less forceful.